lib/compiler.js

```diff
@@ -263,6 +265,36 @@ function sortByInheritance(instructions) {
     });
 }
 
+function verifyModelDefinitions(rootDir, modelDefinitions) {
+  if (!modelDefinitions || modelDefinitions.length < 1) {
+    return undefined;
+  }
+
+  var registry = {};
+  modelDefinitions.forEach(function(definition, idx) {
+    try {
+      var fullPath = path.resolve(rootDir, definition.sourceFile);
+      definition.sourceFile = require.resolve(fullPath);
```

Comment:
This does not support the case when the `sourceFile` value does not have any extension and there are two files matching this value - one is `.json` and the other `.coffee'. See #59 for more details.